fix(user&userGroup): fix bugs about query invalidation and reactivity

seungyeoneeee opened the pull request:
### Skip Review (optional)
- [ ] Minor changes that don't affect the functionality (e.g. `style`, `chore`, `ci`, `test`, `docs`)
- [ ] Previously reviewed in feature branch, further review is not mandatory
- [ ] Self-merge allowed for solo developers or urgent changes

### Description (optional)
- https://mzdevs.atlassian.net/browse/MZ00005-447
- https://mzdevs.atlassian.net/browse/MZ00005-450
- user role update query invalidation
- chore: update for consistent format (showSuccessMessage)

### Things to Talk About (optional)

seungyeoneeee committed:
fix: update bug about user role update query invalidation

Signed-off-by: 이승연 <sylee1274@mz.co.kr>

seungyeoneeee committed:
fix: invalidates queries correctly after worksapce deletion

Signed-off-by: 이승연 <sylee1274@mz.co.kr>

seungyeoneeee committed:
chore: update showSuccessMessage calls for consistent format

Signed-off-by: 이승연 <sylee1274@mz.co.kr>

seungyeoneeee committed:
fix: user group channel schedule form reactivity bugs

Refactors the user group channel schedule form to improve data handling and component reusability.

- Utilizes the store to manage schedule information.
- Employs scoped queries to fetch and display existing schedule data.
- Enhances the update logic for create/update scenarios.

Signed-off-by: 이승연 <sylee1274@mz.co.kr>

yuda110 commented on apps/web/src/services/iam/components/UserManagementTabWorkspace.vue:
```diff
@@ -310,7 +310,9 @@ const { mutateAsync: deleteRoleBinding, isPending: isDeleteRoleBindingPending }
         showSuccessMessage(i18n.t('IDENTITY.USER.MAIN.ALT_S_REMOVE_USER'), '');
         closeRemoveModal();
+        await queryClient.invalidateQueries({ queryKey: roleBindingListQueryKey.value });
         await fetchWorkspaceList();
```
nitpick: ??

yuda110 commented on apps/web/src/services/iam/store/notification-channel-create-form-store.ts:
```diff
@@ -45,6 +45,12 @@ export const useNotificationChannelCreateFormStore = defineStore('channel-create
     });
 
+    const mutations = {
+        updateSchedule(value: UserGroupChannelScheduleInfoType) {
```
quibble: updateSchedule -> updateScheduleInfo 😅

yuda110 commented on apps/web/src/services/iam/components/UserGroupChannelSetInputForm.vue:
```diff
@@ -51,6 +53,25 @@ const { key: notificationProtocolQueryKey, params: notificationProtocolQueryPara
+    })),
+});
+
+const { data: userGroupChannelData } = useScopedQuery({
+    queryKey: userGroupChannelGetQueryKey.value,
+    queryFn: () => userGroupChannelAPI.get(userGroupChannelGetQueryParams.value),
+    enabled: computed(() => !!userGroupPageGetters.selectedUserGroupChannel?.[0]?.channel_id),
+    initialData: () => ({
+        name: '',
+    }),
+    gcTime: 1000 * 60 * 2,
+}, ['DOMAIN', 'WORKSPACE']);
```
suggestion: I recommend creating a custom hook (e.g., `use-user-group-channel-get-query.ts`) to handle this.

seungyeoneeee committed:
fix: update workspace tab vue query method (remove fetch method)

Signed-off-by: 이승연 <sylee1274@mz.co.kr>

seungyeoneeee committed:
chore: update minor bug of not displayed admin role badge

Signed-off-by: 이승연 <sylee1274@mz.co.kr>

seungyeoneeee committed:
chore: renames schedule update method for clarity

Renames `updateSchedule` to `updateScheduleInfo` in the store
and component to better reflect its purpose.

This improves code readability and maintainability.

Signed-off-by: 이승연 <sylee1274@mz.co.kr>

yuda110 reviewed (approved):
LGTM!!!

seungyeoneeee committed:
fix: replace hook of userGroupChannel get query

Signed-off-by: 이승연 <sylee1274@mz.co.kr>
